charts/rstudio/values.yaml

```diff
@@ -9,13 +9,16 @@ AWS:
   IAMRole: ""
   DefaultRegion: "eu-west-1"
 Elasticsearch:
-  Host: "elasticsearch-server"
+  Host: "elasticsearch"
   Port: 9200
+  Scheme: "http"
+  Username: ""
+  Password: ""
 Fluentd:
   Image:
-    Repository: quay.io/fluent/fluentd-kubernetes-daemonset
-    Tag: latest
-    PullPolicy: "IfNotPresent"
+    Repository: fluent/fluentd-kubernetes-daemonset
+    Tag: elasticsearch
```

Comment:
In my case I used the `stable-elasticsearch` tag which at this time should be equivalent to `elasticsearch` but I thought was less likely to break.